website/templates/util/render_nodes.mako

```diff
@@ -1,6 +1,6 @@
 % if len(nodes):
     <ul class="list-group ${'sortable' if sortable and 'write' in user['permissions'] else ''}">
-      <span id="componentScope" class="scripted">
+      <span class="componentScope scripted">
```

Comment:
Changed ID to camelCase as renderNodesList, class name to render-nodes-list for clarity.